pyQuARC/code/custom_validator.py

```diff
@@ -201,3 +219,32 @@ def get_data_url_check(metadata_json):
             "valid": validity,
             "value": value
         }
+
+    @staticmethod
+    def get_data_url_check_umm(related_urls):
+        for url_obj in related_urls:
+            if url_obj.get("Type") == "GET DATA" and (url := url_obj.get("URL")):
+                return {
+                    "valid": True,
+                    "value": url
+                }
+        return {
+            "valid": False,
+            "value": "N/A"
+        }
+
+    @staticmethod
+    @if_arg
+    def url_description_uniqueness_check(related_urls):
```

Comment:
This is very similar to `characteristic_name_uniqueness_check_umm`. Consolidate them and use rules?